src/main/java/org/primefaces/component/breadcrumb/BreadCrumbRenderer.java

```diff
@@ -28,6 +28,7 @@
 import javax.faces.component.UIComponent;
 import javax.faces.context.FacesContext;
 import javax.faces.context.ResponseWriter;
+import org.apache.commons.collections4.keyvalue.DefaultMapEntry;
```

Comment:
We don't use Commons Collections right in PF? I don't think you can use this?

Reply:
nope. how can it compile?

Reply:
It must be part of a provided transitive dependency?

Reply:
yeah from POI. can you please create a issue in general to hide this dependencies in PF?

Reply:
Yep will create an issue.

Reply:
Sorry.. should have been `Map.entry`.. will fix this now